packages/devtools_app/lib/src/standalone_ui/vs_code/debug_sessions.dart

```diff
@@ -143,36 +157,12 @@ class _DevToolsMenu extends StatelessWidget {
         style: const MenuStyle(
           alignment: AlignmentDirectional.bottomStart,
         ),
-        menuChildren: [
-          // TODO(dantup): Ensure the order matches the DevTools tab bar (if
-          //  possible, share this order).
-          // TODO(dantup): Make these conditions use the real screen
-          //  conditions and/or verify if these conditions are correct.
-          devToolsButton(
-            ScreenMetaData.inspector,
-            enabled: isFlutter && isDebug,
-          ),
-          devToolsButton(
-            ScreenMetaData.cpuProfiler,
-            enabled: isDebug || isProfile,
-          ),
-          devToolsButton(
-            ScreenMetaData.memory,
-            enabled: isDebug || isProfile,
-          ),
-          devToolsButton(
-            ScreenMetaData.performance,
-          ),
-          devToolsButton(
-            ScreenMetaData.network,
-            enabled: isDebug,
-          ),
-          devToolsButton(
-            ScreenMetaData.logging,
-          ),
-          // TODO(dantup): Check other screens (like appSize) work embedded and
-          //  add here.
-        ],
+        // TODO(dantup): Why is appSize etc. still missing?
```

Comment:
When I run in the stager app, we use `defaultScreens()` which doesn't include everything (like app size), so I'm sure that's the right thing to use.

Reply:
No long applicable, we're using `ScreenMetaData.values` inside the menu.